### DISCUSSION TITLE
I can't figure out how to implement Loading in nuxt 3

### DISCUSSION THREAD

**Discussion by unknownman** [nuxt/nuxt#17547] [Questions]
Considering the use of composition api in useFetch and considering that loading management was possible in Nuxt v2; Is there a plan for a comprehensive loading management in the new version?

**Comment by TheAlexLichter** [comment]
Can you elaborate your issue a little?

WIth `useFetch`, you also have a `pending` ref exposed that you can use to find out whether the request is ongoing or not ☺️ 
See [the docs](https://v3.nuxtjs.org/api/composables/use-fetch#type)

**Comment by unknownman** [comment]
in nuxt 2 there is a loading property that we are able to manage loading with it in a Page : https://main--nuxt.netlify.app/docs/features/loading/

or for example we there is a loading plugin in Quasar that can be modified through nuxt.config

my question is : Are there  any plan for developing a universal plugin for loading or we must handle it with pending ref

**Comment by TheAlexLichter** [comment]
See also https://github.com/nuxt/framework/discussions/2100 and https://github.com/nuxt/framework/pull/5121 ☺️
